Geometry/CMSCommonData/python/cmsExtendedGeometry2017aXML_cfi.py

```diff
@@ -0,0 +1,287 @@
+import FWCore.ParameterSet.Config as cms
+
+## 2015 + new phase 1 pixel detector
+
+XMLIdealGeometryESSource = cms.ESSource("XMLIdealGeometryESSource",
```

Comment:
This is yet another scenario for HCAL so called Plan-1 where only one RBX of HE is upgraded. The current 2017 scenario is so called Plan-0 where only HF is upgraded and HE is not. The current 2018 scenario is Plan-36 when all 36 RBX's of HE are upgraded.